export/http_test.go

```diff
@@ -54,7 +54,7 @@ func TestHTTPExportMetrics(t *testing.T) {
 
 		want, err := proto.Marshal(metricsRequest)
 		if err != nil {
-			log.Fatal("Marshalling error: ", err)
+			t.Errorf("Marshalling error: %v", err)
```

Comment:
if you change log.Fatal() to t.Errorf(), then you will need to a "return" after t.Errorf(). Otherwise the code would continue, and `want` could be null afterwards